focus/commands/src/cli/main.rs

```diff
@@ -494,6 +495,18 @@ enum IndexSubcommand {
         sparse_repo: PathBuf,
     },
 
+    /// Calculate statistics on cache invalidation for the projects in the
+    /// repository.
+    CalculateChurn {
+        /// Path to the sparse repository.
+        #[clap(parse(from_os_str), default_value = ".")]
```

Comment:
You might want to make clap handle this as "long". I am going to go through and change all of these positional arguments soon unless you can think of a good reason not to.